lib/pagerbot/plugin/call_person.rb

```diff
@@ -14,7 +14,7 @@ def initialize(config)
     def self.manual
       {
         description: "Page a person by nick or schedule (which could be a team)",
-        syntax: ["get PERSON|SCHEDULE [SUBJECT]"],
+        syntax: ["get TEAM_OR_SCHEDULE [SUBJECT]"],
```

Comment:
let's keep this `TEAM|SCHEDULE` to be consistent with other messages. `|` is often used as a visual representation of Logical OR in command-line help messages